client/src/components/Employee/EmployeeListing.js

```diff
@@ -28,12 +32,8 @@ const EmployeeListing = props => {
   const [userToUpdate, setUserToUpdate] = useState()
   const [updateAction, setUpdateAction] = useState()
   const [showDelDialog, setShowDelDialog] = useState(false)
-  const deleteEmployeeSuccess = useSelector(
-    state => state.EmployeeInfo.deleteEmployeeSuccess
-  )
-  const deleteEmployeeError = useSelector(
-    state => state.EmployeeInfo.deleteEmployeeError
-  )
+  const deleteEmployeeSuccess = useSelector(state => deleteEmployeeSuccessMsg(state))
```

Comment:
Can you try using  `useSelector(deleteEmployeeSuccessMsg)` ?

Reply:
Done